scripts/genesis-tools.Dockerfile

```diff
@@ -10,14 +10,14 @@
 # The builder is based on a base image that has stack, haskell, and rust installed.
 # The only argument is the branch name of a concordium-base repository branch, given as BASE_REF.
 
-FROM 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/base:0.18 AS builder
+FROM concordium/base:0.19 AS builder
 
 # Which branch of concordium-base to build the tools from.
-ARG BASE_REF=master
+ARG BASE_REF=main
 
-RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan gitlab.com >> ~/.ssh/known_hosts
+RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
 
-RUN --mount=type=ssh git clone --depth 1 --branch ${BASE_REF} git@gitlab.com:Concordium/concordium-base.git
+RUN --mount=type=ssh git clone --depth 1 --branch ${BASE_REF} git@github.com:Concordium/concordium-base.git
```

Comment:
`git clone --depth 1 --branch main https://github.com/Concordium/concordium-base.git` would be sufficient.

Reply:
(can't make multiline suggestions)

Reply:
I can change. 

But the reason I did not is that there might be a case where we'll need a private fork for security fixes, in which case we'll need this again.

Are you strongly against keeping it like this?

Reply:
Not strongly against it, no.

Reply:
Ok, then I'll keep it if only for myself to not forget how to do this in the future.